### PR TITLE
Project 8: Popup window

### PR DESCRIPTION
**Pull request by yaseerokino**
## Features Added ♻️ 

- Added popup modal section for each project
- Updated files structure

### CODE REVIEW

**Review by Davidosky007 — CHANGES_REQUESTED** [review]
# Hi, @yaseerokino @mbdesigns1989 👋, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

File: js/main.js [hunk 1]
```diff
@@ -9,5 +66,20 @@ const closeMenu = () => {
...
+};
+
+const closeModal = () => {
+  body.classList.remove('no-scroll');
+  modal.style.display = 'none';
+};
 document.getElementById('openbutton').addEventListener('click', openMenu);
 document.getElementById('closebutton').addEventListener('click', closeMenu);
```
[the remark this answers is not in the document]
Comment (yaseerokino) [reply js/main.js]: Resolved

File: js/main.js [hunk 2]
```diff
@@ -1,5 +1,62 @@
...
+    categories: ['PRINTERS', 'Front End Dev', '2015'],
+    image: {
+      src: 'images/work1.png',
+      alt: 'dolist',
+    },
+    description:
+      'A daily selection of privately personalized reads; no accounts or sign-ups required.',
+    technologies: ['React', 'Node', 'Express', 'MongoDB', 'Mongoose', 'Axio'],
+    liveLink: 'https://yaseerokino.github.io/personal-portfolio/',
+    sourceLink: 'https://github.com/yaseerokino/personal-portfolio',
```
[the remark this answers is not in the document]
Comment (yaseerokino) [reply js/main.js:36]: resolved

[the remark this answers is not in the document]
Comment (dasileker) [reply js/main.js:36]: - [ ] Please don't remove the button for the Github repo's
card instead use a div to make it even.

**Review by dasileker — CHANGES_REQUESTED** [review]
### **Hi @yaseerokino** 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: js/main.js (hunk 2 above)
[the remark this answers is not in the document]
Comment (yaseerokino) [reply js/main.js:36]: @dasileker the image above is not the modal image and that of the portfolio section. And that is how the design is on the Figma template

**Review by dasileker — APPROVED** [review]
### **Hi @yaseerokino** 

### **Project approved ✅✅**

### to highlight

- The README file is pro.
- Codes are clean

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**